project-reports/2024/2024-annual-Hyperledger-Bevel.md

```diff
@@ -0,0 +1,87 @@
+4. Bevel recently came out with its general availability release v1.0, with focus on seamless deployment. 
+5. Bevel-Operator-Fabric recently came out with its release v1.10.0, with a focus on stability of network deployment.
+6. Bevel has accepted a major code contribution. (helm charts for deploying Hyperledger Aries).
+7. The projects are on track with its current roadmap and plans for the FY.
+
+
+# Maintainer Diversity
+
+-	There is no change in the maintainers for Bevel. You can find our current maintainer list here:  
+<a href="https://github.com/hyperledger/bevel/blob/main/MAINTAINERS.md" class="external-link" rel="nofollow">https://github.com/hyperledger/bevel/blob/main/MAINTAINERS.md</a>.
+
+<!-- -->
+
+-	Existing maintainers:
+	-	Jagpreet Singh Sasan (Accenture)
+	-	Sownak Roy (Accenture)
+	-	Suvajit Sarkar (Accenture)
+	-	Aditya Joshi (Walmart)
+	-	Arun S M (Walmart)
+	-	Weihong Ou (Walmart)
+-	Emeritus Maintainers:
+	-	Jonathan M Hamilton (Accenture)
+	-	Tracy Kuhrt (Accenture)
+
+-	There is no change in the maintainers Bevel-Operator-Fabric. You can find our current maintainer list here:  
+<a href="https://github.com/hyperledger/bevel-operator-fabric/blob/main/MAINTAINERS.md" class="external-link" rel="nofollow">https://github.com/hyperledger/bevel-operator-fabric/blob/main/MAINTAINERS.md</a>.
+
+<!-- -->
+
+-	Existing maintainers:
+	-	David Viejo (Kung Fu Software)
+	-	Aditya Joshi (Walmart)
+
+
+# Project Adoption
+
+Based on what we see in the mailing list and discord channels there is significant adoption activity.
+Few large vendors and several smaller companies are using Hyperledger Bevel for DLT network automation.
+
+# Goals
+
+## Performance Against Prior Goals
+
+### Goal: Major Features
+- Bevel general availability release v1.0 (completed)
+- Bevel Documentation restructure and redesign (completed)
+- Fabric v2.5.x support on both projects (completed)
+
+### Goal: Growing the Community
+- Blog post on Bevel release v1.0 (published)
+- Hyperledger meetup featuring Bevel release v1.0 (50+ participants)
+- Workshop on Bevel and Bevel-Operator-Fabric (90+ participants) 
+- Hyperledger meetup featuring Bevel Fabric Operator supporting fabric 3.0
+
+## Next Year's Goals
+
+### Goal 2024: Major Features
+- Network deployments using just helm charts
+- Add support for cloud managed vaults
+- Use of Kubernetes operator for network orchestration
+- Stability of the operator and support of Fabric v3.0 using bevel-operator-fabric
+
+### Goal 2024: Growing the Community
+- Workshop and demos on upcoming changes and features
+
+## Help Required
+
+- Continue to work with Hyperledger staff to identify contributors and potential maintainers outside current maintainers and orgs.
+- Mention Hyperledger Bevel as a means for deployment (on Kubernetes) on documentations’ of Hyperledger projects such as Fabric, Besu, Indy and Aries.
+
+# Project Lifecycle Stage Recommendation
+Currently in `incubation`, has potential to be `Graduated`.
```

Comment:
Is this a request/recommendation to move to graduated or more of an observation?  It sounds like the latter.

Reply:
Yeah now that you have asked, it does sounds like an observation. But what we meant to convey is that, we need to do some more work (one more major release) before we can confidently request/recommend for the project graduation.